examples/cisco/demo_evpn_multisite.pp

```diff
@@ -0,0 +1,28 @@
+# Manifest to demo evpn_multisite providers
+#
+# Copyright (c) 2017 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+class ciscopuppet::cisco::demo_evpn_multisite {
+
+  if platform_get() =~ /n9k-ex/ {
```

Comment:
This function will need to be updated to return a value if it's an n9k-ex platform.

https://github.com/cisco/cisco-network-puppet-module/blob/develop/lib/puppet/parser/functions/platform_get.rb

Reply:
Had completely missed this. Thanks for pointing it out.